.gitignore

```diff
@@ -0,0 +1,2 @@
+/vendor/*
+.idea
```

Comment:
.idea it's not a single file it's a folder. Please change it to exclude all files within in